src/errors.rs

```diff
@@ -28,12 +28,14 @@ pub enum Error {
     #[error("Inner type cannot be converted to requested type")]
     MismatchType,
 
-    #[error(transparent)]
-    ReqwestError(#[from] reqwest::Error),
+    // #[error(transparent)]
+    // ReqwestError(#[from] reqwest::Error),
 
+    // FIXME: Remove, already in NetworkError enum
     #[error(transparent)]
     TungsteniteError(#[from] tungstenite::Error),
 
+    // FIXME: Remove???, already in NetworkError enum -- maybe it requires the refactor to isolate all http stuff from the client library
     #[error("Protocol error. Unexpected data received from server")]
     ProtocolError(String),
```

Comment:
In the same comment as the one for the `NetworkError`: 

This `ProtocolError` is duplicated in this `Error` enum... we are raising it from [here](https://github.com/IBM/appconfiguration-rust-sdk/blob/a6615ce7a9a6509fcfab505921dada1b01ea417b/src/models.rs#L160) as well. I guess we will need the refactors that we have started in #56 in order to isolate it into the network module.